docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+```
+peer channel getinfo -c channel1
+```
+The command will list the block height of the channel and the hash of the most recent block. Because the genesis block is the only block on the channel, the height of the channel will be one:
+```
+2020-03-13 10:50:06.978 EDT [channelCmd] InitCmdFactory -> INFO 001 Endorser and orderer connections initialized
+Blockchain info: {"height":1,"currentBlockHash":"kvtQYYEL2tz0kDCNttPFNC4e6HVUFOGMTIDxZ+DeNQM="}
+```
+
+We can now join the Org2 peer to the channel. Set the following environment variables to operate the `peer` CLI as the Org2 admin. The environment variables will also set the Org2 peer, ``peer0.org1.example.com``, as the target peer.
+```
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_LOCALMSPID="Org2MSP"
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
+export CORE_PEER_ADDRESS=localhost:9051
+```
+
+While we still have the channel genesis block on our file system, in a more realistic scenario, Org2 would have the fetch the block from the ordering service. As an example, we will use the `peer channel fetch` command to get the genesis block for Org2:
+```
+peer channel fetch 0 ./channel-artifacts/channel_org2.block -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com -c channel1 --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+
+The command uses `0` to specify that it needs to fetch the genesis block that is required to join the channel. If the command is successful, you should see the following output:
+```
+2020-03-13 11:32:06.309 EDT [channelCmd] InitCmdFactory -> INFO 001 Endorser and orderer connections initialized
+2020-03-13 11:32:06.336 EDT [cli.common] readBlock -> INFO 002 Received block: 0
+```
+
+The command returns the channel genesis block and names it `channel_org2.block` to distinguish it from the block pulled by org1. You can now use the block to join the Org2 peer to the channel:
+```
+peer channel join -b ./channel-artifacts/channel_org2.block
+```
+
+## Set anchor peers
+
+After an organizations has joined their peers to the channel, they should select at least one of their peers to become an anchor peer. Anchor peers are used to communicate with other peers on the channel using gossip. Anchor peers disseminate new blocks to ensure that all peers in your organization have the most recent data. They are also used by client applications to discover information about the channel. It is required to set anchor peers in order to take advantage of features such as private data and service discovery. For more information about gossip and anchor peers, see the [Gossip data dissemination protocol](../gossip.html).
```

Comment:
After an organization has joined their peers...

[Anchors peers](https://hyperledger-fabric.readthedocs.io/en/latest/gossip.html#anchor-peers) are used to communicate with peers in other organizations on the channel.

Reply:
done.